### DISCUSSION TITLE
Multiple books (in different folders) mixed together into one audio book

### DISCUSSION THREAD

**Discussion by juriroemer** [advplyr/audiobookshelf#143] [Q&A]
I have two folders of the structure
```
- author
    - series
        - book 1
            - audiofiles ...
        - book 2
            - audiofiles ...
        - book 3
             - audiofiles ...
        - …
```
where the books don't get recognized as individual books, but get thrown together into one book (which gives all kinds of problems). These books are a very loose series so i also tried 
```
- author
    - book 1
        - audiofiles ...
    - book 2
        - audiofiles ...
    - book 3
        - audiofiles ...
    - …
```
which gave me the exact same result. Has anyone had this problem, or an idea of what could be going on? I looked at the metadata and couldn't see anything that would confuse audiobookshelf in this way, but I may have missed something.

EDIT: I managed to work around this by adding the books one by one instead of adding the ready to do folder structure into the audiobooks folder. Still not sure why that didn't work though.

**Comment by advplyr** [comment]
This was reported the other day also. I'm not sure where to look to try to debug this since we haven't narrowed down what is happening.

One thing that could be happening that may not be clear in the docs, is that if an ebook file (pdf included) or an audio file exists in your author or series folder, then it will consider the author or series folder as a title folder.

The scanner is looking for audio files and ebooks, when it finds one it assumes that the folder that it's in is a title folder.

Next thing to look out for is if the folder has audio files but the scanner could not probe them or wasn't able to identify any track numbers, then the folder gets ignored. So if you have a folder with multiple audio files that have no metadata and have no number in the filename, then it won't get scanned in. This is something that still needs to get fixed.

**Comment by jarrodCoombes** [comment]
I have had this happen to me as well. With little to no explanation. To fix it I just removed the "book" changed the series/author folder name, then changed it back. Seemed to work in most cases. A few times I had to change the parent folder name and let the scanner do it's thing and either left it or changed it back.
